src/containers/WelcomePage/components/Revisions.tsx

```diff
@@ -70,8 +70,8 @@ const StyledTimeIcon = styled(Time)`
   &[data-status='expired'] {
     fill: ${colors.support.red};
   }
-  width: ${spacing.snormal};
-  height: ${spacing.snormal};
+  width: 20px;
```

Comment:
Kunne kanskje hatt en constants i core som vi kunne brukt?

Reply:
Har vi nok greier som ska ha faste størrelser til å f.eks. ha en size-del i core? Kunne f.eks. vært strukturert size.icon.small, size.icon.medium osv (og ev. andre ting som ska ha størrelser). Jeg ser jo at det er noe størrelser i bruk alt, andre steder, men kanskje sizing kunne vært samlet, og dermed også samla dette?

Reply:
Jeg har mer lyst til en streng styring av størrelsene av icons, litt mer likt slik vi gjør med buttons. Det synes jeg hadde vært knall. Da har vi kun et sted å måtte passe på 👯

Reply:
Enig med Lars, men synes det heller er noe vi kan gjøre når vi jobber med nytt designsystem/profil.

Reply:
Det var det jeg mente (men kanskje forklarte dårlig). Det er egentlig det samme for meg om det løses med sizes i core, eller ved hjelp av dataattributt på selve ikonet. Men vi har vel behov for minst 2 ikonstørrelser om vi sku skrevet om ed, f.eks. (hvor ikoner er brukt både i header og inline i tabellene på dashboard).

Reply:
Lag et issue på det!